### PR TITLE
Added `attributes: :accessors` option to `define_attributes`

### PR DESCRIPTION
**Pull request by pnomolos**
- Slightly slower than using ivars, hence the need for a new option, but allows you to override the getters and still have them returned via the `attributes` call.

### CODE REVIEW

File: spec/fast_attributes_spec.rb [hunk 1]
```diff
@@ -363,5 +363,26 @@ def object.to_str; 1/0 end
...
       end
     end
+
+    describe 'option attributes: :accessors' do
+      it 'generates attributes method' do
+        publisher = Publisher.new
+        expect(publisher.attributes).to eq({'name' => nil, 'books' => nil})
+
+        reader = Reader.new
+        expect(reader.attributes).to eq({'name' => nil, 'age' => nil})
```
Comment (kostyantyn) [inline spec/fast_attributes_spec.rb]: Could you, please, use in this test `AttributesWithAccessors` class which has option `: accessors` set

**Comment by pnomolos** [comment]
@kostyantyn I'll get this PR completed so it's ready before you push a new release :)

**Comment by kostyantyn** [comment]
Thanks again!

**Comment by kostyantyn** [comment]
Version 0.9.0 is released

**Comment by psychocandy** [comment]
🎉 ✨

**Pull request merged**